samples/getting-started/qrng/Qrng.qs

```diff
@@ -16,18 +16,25 @@ namespace Qrng {
     }
     
     @EntryPoint()
-    operation Main() : Int {
+    operation RunMain() : Int {
         let max = 10;
```

Comment:
What about max = 50? For uniformity with the host codes and also it gets the probability of failing a bit lower (~20% instead of 33%)

Reply:
Good point, let me go on and make the standalone version more uniform with the interoperability version.